openmdao/components/tests/test_external_code_comp.py

```diff
@@ -610,13 +613,17 @@ def setup(self):
                 self.options['external_input_files'] = [self.input_file]
                 self.options['external_output_files'] = [self.output_file]
 
+
                 self.options['command_apply'] = [
                     sys.executable, 'extcode_mach.py', self.input_file, self.output_file,
                 ]
                 self.options['command_solve'] = [
                     sys.executable, 'extcode_mach.py', self.input_file, self.output_file,
                 ]
 
+                # If you want to write your own string command, the code below will also work.
+                # self.options['command_apply'] = 'python extcode_mach.py mach_input.dat mach_output.dat'
+
```

Comment:
I don't think you should hard-code the file names here, since they are specified via attributes.
Should probably do something like:
_cmd_ = `python extcode_mach.py {} {}'.format(self.input_file, self.output_file)`